pkg/statistics/handle/bootstrap.go

```diff
@@ -229,13 +230,19 @@ func (h *Handle) initStatsHistograms4Chunk(is infoschema.InfoSchema, cache stats
 }
 
 func (h *Handle) initStatsHistogramsLite(is infoschema.InfoSchema, cache statstypes.StatsCache) error {
-	ctx := kv.WithInternalSourceType(context.Background(), kv.InternalTxnStats)
 	sql := "select HIGH_PRIORITY table_id, is_index, hist_id, distinct_count, version, null_count, tot_col_size, stats_ver, correlation, flag, last_analyze_pos from mysql.stats_histograms"
 	rc, err := util.Exec(h.initStatsCtx, sql)
 	if err != nil {
 		return errors.Trace(err)
 	}
 	defer terror.Call(rc.Close)
+	if config.GetGlobalConfig().Performance.ConcurrencyInitStats {
+		ls := loadstats.NewLoadStats(rc.Next, h.initStatsHistograms4ChunkLite)
+		ls.LoadStats(is, cache, rc)
+		ls.Wait()
+		return nil
+	}
+	ctx := kv.WithInternalSourceType(context.Background(), kv.InternalTxnStats)
```

Comment:
Please add // TODO comment to remove those non-concurrent load stats logic in v8.5

Reply:
We can enable it by default at v8.5. and remove it at v9.1.